plugins/repository-s3/src/main/java/org/elasticsearch/repositories/s3/InternalAwsS3Service.java

```diff
@@ -75,12 +75,6 @@ public synchronized AmazonS3 client(Settings repositorySettings) {
                 Strings.collectionToDelimitedString(clientsSettings.keySet(), ","));
         }
 
-        Integer maxRetries = getValue(repositorySettings, settings,
-            S3Repository.Repository.MAX_RETRIES_SETTING,
-            S3Repository.Repositories.MAX_RETRIES_SETTING);
-        boolean useThrottleRetries = getValue(repositorySettings, settings,
-            S3Repository.Repository.USE_THROTTLE_RETRIES_SETTING,
-            S3Repository.Repositories.USE_THROTTLE_RETRIES_SETTING);
         // If the user defined a path style access setting, we rely on it,
         // otherwise we use the default value set by the SDK
         Boolean pathStyleAccess = null;
```

Comment:
You wrote in the PR description:

>  The path style access setting should be removed altogether.

But you did not change it. Does it mean that you are planning to do that in another PR?